app/zpt/handler/atmos.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from scipy.integrate import odeint  # type: ignore
+from scipy.interpolate import BSpline, splrep  # type: ignore
+
+from .geos import g  # type:ignore
+
+AVOGADRO = 6.02282e23  # [mol^-1] aovogadros number
+
+
+def intermbar(z):
+    mbars = np.r_[
```

Comment:
`np.r_[...]` in this case only does the exact same thing `np.array([...])` does. I think the latter looks a little less obscure. It also seems a bit wasteful to reinitialize `mbars` and `mbarz` every time the function is called, but maybe it is only called once?